cfgov/cfgov/settings/base.py

```diff
@@ -134,7 +135,6 @@
 
 # Sheer related settings
 
-SHEER_SITES = [Path(os.path.dirname(__file__)).ancestor(2) + '/v1/jinja2/v1',
-               Path(__file__).ancestor(4).child('docs'), STATIC_ROOT]
+SHEER_SITES = [V1_TEMPLATE_ROOT, Path(__file__).ancestor(4).child('docs'), STATIC_ROOT]
```

Comment:
hmmm we could, looked like it had some useful template convention related information. i guess it doesn't hurt to leave it in?